### ISSUE TITLE
"Show Images" icon invisible if "Dark Theme" is enabled in "Accessibility" settings

### ISSUE THREAD

**Issue by klassikmann** [nextcloud/mail#4660]
### Expected behavior
"Dark Theme" is enabled in "Accessibility" settings. The "Show Images" icon should be visible in mails where images have been blocked.

### Actual behavior
The icon is invisible, exception: you hover the mouse over icon position.

### Mail app
1.9.1

**Mailserver or service:** (e.g. Outlook, Yahoo, Gmail, Exchange,...)
T-Online

### Server configuration

**Operating system**: 
OMV5 on Armbian 21.02.2 Buster

**Web server:**
nginx

**Database:** 
MariaDB

**PHP version:** (e.g. 7.0)
7.4.15

**Nextcloud Version:**
20.0.8

#### Client configuration
**Browser:**
Firefox 86.0

**Operating system:**
Windows 10 20H2

**Comment by ChristophWurst** [comment]
Mind sharing a screenshot? :)

**Comment by klassikmann** [comment]
> Mind sharing a screenshot? :)

I have to wait until a new appropriate mail arrives.

**Comment by klassikmann** [comment]
Here we are:
1. "Dark theme" unselected, "show image" icon visible: 
![pic2](https://user-images.githubusercontent.com/25383513/109969563-57183200-7cf4-11eb-952e-2531c69a8a70.jpg)
2. "Dark theme" selected, "show image" icon invisible:
![pic1](https://user-images.githubusercontent.com/25383513/109969637-6dbe8900-7cf4-11eb-82ad-1e98be05095f.jpg)
3. "Dark theme" selected, "show image" visible, if mouse pointer over image:
![pic3](https://user-images.githubusercontent.com/25383513/109969786-9777b000-7cf4-11eb-9ed8-5e622e9b6a03.jpg)

**Comment by ChristophWurst** [comment]
Regression of https://github.com/nextcloud/mail/pull/4376 or an issue with https://github.com/nextcloud/mail/pull/4513

:eyes: @GretaD

**Comment by klassikmann** [comment]
fyi
The issue has not been resolved in 1.9.3.

**Comment by ChristophWurst** [comment]
> fyi
> The issue has not been resolved in 1.9.3.

Yes, because the fix wasn't backported.

**Comment by klassikmann** [comment]
Resolved in 1.9.4.
